OSDOCS-16949#CQA work Stor5 - CSI IBM Power Block Driver (#114609)
openshift/openshift-docs: modules/persistent-storage-csi-ibm-powervs-block-overview.adoc

// Module included in the following assemblies:
//
// * storage/container_storage_interface/persistent-storage-csi-imb-powervs.block.adoc

:_mod-docs-content-type: CONCEPT
[id="persistent-storage-csi-ibm-powervs-block-overview_{context}"]

> = Overview

Reviewer: need a meaningful title here

[role="_abstract"]
{product-title} can provision persistent volumes (PVs) by using the Container Storage Interface (CSI) driver for {ibm-power-server-name} Block Storage.

> Familiarity with persistent storage and configuring CSI volumes is helpful when working with a CSI Operator and driver. Fore more information, see _Understanding persistent storage_ and _Configuring CSI volumes_ under _Additional resources_.

Reviewer: current style is quotes, not itals

To create CSI-provisioned PVs that mount to {ibm-power-server-name} Block storage assets, {product-title} installs the {ibm-power-server-name} Block CSI Driver Operator and the {ibm-power-server-name} Block CSI driver by default in the `openshift-cluster-csi-drivers` namespace.

* The _{ibm-power-server-name} Block CSI Driver Operator_ provides two storage classes named `ibm-powervs-tier1` (default), and `ibm-powervs-tier3` for different tiers that you can use to create persistent volume claims (PVCs). The {ibm-power-server-name} Block CSI Driver Operator supports dynamic volume provisioning by allowing storage volumes to be created on-demand, eliminating the need for cluster administrators to pre-provision storage.

> * The _{ibm-power-server-name} Block CSI driver_ allows you to create and mount {ibm-power-server-name} Block PVs.

Reviewer: ```suggestion
* By using the _{ibm-power-server-name} Block CSI driver_  you can create and mount {ibm-power-server-name} Block PVs.
```
IBM does not prefer "allows" (and this removes passive voice)
